pixart.js

```diff
@@ -1 +1,28 @@
+$(document).ready(function() {
+var colorSelect;
 
+$('#set-color').click(color)
```

Comment:
nice! it might be easier to read with a more semantic function name, like `setColor`